tests/unit/src/test/resources/foldingRange/expect/Main.java

```diff
@@ -1,10 +1,16 @@
 package example;
 
->>imports>>import org.eclipse.jdt.core.ToolFactory;
-import org.eclipse.jdt.core.compiler.ITerminalSymbols;
-
-import org.eclipse.lsp4j.FoldingRange;
-import org.eclipse.lsp4j.FoldingRangeKind;<<imports<<
+>>imports>>import java.io.File;
```

Comment:
Last thing, could add the test to `foldingRange-scala3-foldLineOnly` folder also? It should handle Java properly (though it would probably better to create a separate suite, but let's not do it now.) This way we'll see the difference between fold only on and off

Reply:
Done